cloudslang-compiler/src/main/java/io/cloudslang/lang/compiler/modeller/transformers/WorkerGroupTransformer.java

```diff
@@ -0,0 +1,85 @@
+/*******************************************************************************
+ * (c) Copyright 2016 Hewlett-Packard Development Company, L.P.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Apache License v2.0 which accompany this distribution.
+ *
+ * The Apache License is available at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *******************************************************************************/
+package io.cloudslang.lang.compiler.modeller.transformers;
+
+import io.cloudslang.lang.compiler.CompilerConstants;
+import io.cloudslang.lang.compiler.SlangTextualKeys;
+import io.cloudslang.lang.compiler.modeller.result.BasicTransformModellingResult;
+import io.cloudslang.lang.compiler.modeller.result.TransformModellingResult;
+import io.cloudslang.lang.entities.SensitivityLevel;
+import io.cloudslang.lang.entities.WorkerGroupStatement;
+import org.apache.commons.lang.StringUtils;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class WorkerGroupTransformer extends AbstractInOutForTransformer
```

Comment:
inline

Reply:
Can't inline because the line will be too long and the code style checker maven plugin will complain at compile time